### ISSUE TITLE
All servers crash with latest commits (bt included)

### ISSUE THREAD

**Issue by chriser-** [rathena/rathena#178]
```C++
Program received signal SIGSEGV, Segmentation fault.
0x00000000005c3a2b in db_alloc (file=0x5e55cc "map.c", line=4259, type=DB_INT, 
    options=DB_OPT_BASE, maxlen=4) at db.c:2445
2445			db->ht[i] = NULL;
(gdb) bt full
#0  0x00000000005c3a2b in db_alloc (file=0x5e55cc "map.c", line=4259, 
    type=DB_INT, options=DB_OPT_BASE, maxlen=4) at db.c:2445
        db = 0x7ffff6708f4c
        i = <optimized out>
        __FUNCTION__.4687 = "db_alloc"
#1  0x00000000004bc700 in do_init (argc=<optimized out>, argv=<optimized out>)
    at map.c:4259
No locals.
#2  0x0000000000409fe5 in main (argc=1, argv=0x7fffffffe458) at core.c:359
No locals.
```

Happended somewhere from 78bcd8203929318c85c601e3717529eefb06e2b2 to 36717c6a35db0b89140e31980a062ad47f4fcf4e
Every server (login, char, map) has the same backtrace.

**Comment by lukyrys** [comment]
same issue as mine - https://github.com/rathena/rathena/issues/193

**Comment by chriser-** [comment]
Fix is included in #193
